OJP/OJP_Trips.xsd

```diff
@@ -743,6 +743,38 @@
 					<xs:documentation>Number how often the access feature occurs in this PathLink</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="From" type="PathLinkEndStructure" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Designations of the level where the PathLink starts.</xs:documentation>
```

Comment:
Designations of level and place where this PathLink starts?

Reply:
@sgrossberndt Your're right - this is a better description of the structure's purpose since the addition of the Id attribute.

Reply:
changed